definitions/npm/react-with-separator_v1.x.x/flow_v0.72.x-/react-with-separator_v1.x.x.js

```diff
@@ -0,0 +1,10 @@
+declare module "react-with-separator" {
+  import type { ChildrenArray } from "react";
```

Comment:
I'm pretty sure this will be typed as `any`, can you use the internal version?

Reply:
AFAIK there is no internal version for `ChildrenArray`; I did try to do that first.  There is an issue to fix this - https://github.com/facebook/flow/issues/4787

But given that `cloneElement` doesn't accept this type, I think this is now moot.